dev-util/kdstatemachineeditor/kdstatemachineeditor-1.2.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions multilib
+
+QT_MINIMAL="5.3"
+
+DESCRIPTION="Framework for creating Qt State Machine metacode using graphical user interfaces"
+HOMEPAGE="https://github.com/KDAB/KDStateMachineEditor/ \
+	https://github.com/KDAB/KDStateMachineEditor/wiki/"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/KDStateMachineEditor.git"
+else
+	SRC_URI="https://github.com/KDAB/KDStateMachineEditor/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+	if [[ ${PV} == 1.2.1 ]] ; then
```

Comment:
Unecessary conditional. Move the PATCHES array down below DEPEND where it belongs and just remove it from 9999.

Reply:
Patch should apply only for `${PV}` = 1.2.1. They fixed their [CMakeLists.txt](https://github.com/KDAB/KDStateMachineEditor/commit/e6ac7f3ca0ed0fe699d63eba822b94f8486a7e7e), new versions won't need the patch.

Maybe a conditional in `src_prepare()` is a better approach?

Reply:
no conditional at all. why bother put it in the 9999 ebuild at all if you only need it in 1.2.1?

Reply:
@a17r It turns out there isn't need for a patch; Version 1.2.3 is already out and [doesn't](https://github.com/KDAB/KDStateMachineEditor/blob/master/CMakeLists.txt#L95) need it.

Reply:
even better then.